src/components/PaymentCard.tsx

```diff
@@ -459,7 +459,7 @@ export const PaymentCard = ({
                 <Button
                   size="small"
                   variant="outlined"
-                  onClick={() => openExplorer(asset.address)}
```

Comment:
`asset` is passed to `useAsset hook, and return `tokenAddress`. Is this hook is still needed?